### PR TITLE
feat - implement footer

### PR DESCRIPTION
**Pull request by seanenright**
To calculate the footer summary I was looking looking at what these other projects did:
https://github.com/gitgrimbo/harviewer-react
https://github.com/janodvarko/harviewer

Chrome dev-tools uses different terms for compressed and uncompressed data. I don't know if we want to use Google's terminology or just match the existing network viewer. The only thing missing is finished time.  We are missing 'FinishedTime' - I'm not sure it's important (yet).  But it looks to be similar to a cpuIdle or totalTracingTime.
[x]Total Requests 
[x]DOM Content Loaded
[x]Load Time
[x]Data transferred (compressed data) chome
[x] Resources (uncompressed data)

- Finished Time

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -208,3 +256,29 @@ export const findIndexByTimeStamp = (data, exactTimestamp) => (
...
+  }
+  if (bytes < (1024 * 1024)) {
+    return `${Math.round(bytes / 1024)}KB`;
+  }
+  return `${Math.round((bytes / (1024 * 1024)))}MB`;
+};
+
+export const formatTime = (time) => {
+  if (time < 1000) {
+    return `${time}ms`;
```
Comment (farhan-sauce) [inline src/utils.js]: Screen Shot 2020-01-07 at 2 31 12 PM

can we use `Math.round` here?

File: src/utils.js [hunk 2]
```diff
@@ -56,30 +56,78 @@ export const getContent = ({ mimeType, text }) => {
...
+  if (_transferSize > -1) {
+    return _transferSize;
+  }
+  if (bodySize > -1) {
+    return bodySize;
+  }
+  return -1;
+};
+
+export const isCachedEntry = ({ response }) => {
```
Comment (farhan-sauce) [inline src/utils.js]: Do we use this function anywhere?

File: src/utils.js [hunk 3]
```diff
@@ -208,3 +256,29 @@ export const findIndexByTimeStamp = (data, exactTimestamp) => (
...
+  pages.reduce(({ DOMContentLoaded, onLoad }, { pageTimings }) => ({
+    DOMContentLoaded: DOMContentLoaded + pageTimings.onContentLoad,
+    onLoad: onLoad + pageTimings.onLoad,
+  }), { DOMContentLoaded: 0, onLoad: 0 }));
+
+export const formatSize = (bytes) => {
+  if (bytes < 1024) {
+    return `${bytes}B`;
+  }
+  if (bytes < (1024 * 1024)) {
```
Comment (farhan-sauce) [inline src/utils.js]:
```suggestion
  if (bytes < (1024 ** 2)) {
```

File: src/utils.js [hunk 4]
```diff
@@ -208,3 +256,29 @@ export const findIndexByTimeStamp = (data, exactTimestamp) => (
...
+    return `${Math.round(bytes / 1024)}KB`;
+  }
+  return `${Math.round((bytes / (1024 * 1024)))}MB`;
```
Comment (farhan-sauce) [inline src/utils.js]:
```suggestion
  return `${Math.round((bytes / (1024 ** 2)))}MB`;
```

File: src/utils.js [hunk 5]
```diff
@@ -56,30 +56,78 @@ export const getContent = ({ mimeType, text }) => {
...
+
+  if (bodySize > -1) {
+    return bodySize;
+  }
+  return -1;
+};
+
+export const getEntryUncompressedSize = ({ response }) => {
+  const { bodySize, _transferSize, content: { size } } = response;
+  if (size > -1) {
```
Comment (farhan-sauce) [inline src/utils.js]: By changing `if (size > 0) {` this function will return size of resources.

**Comment by seanenright** [comment]
@farhan-sauce okay now?

**Pull request merged**